src/LibraryViewExtension/Handlers/NodeItemDataProvider.cs

```diff
@@ -84,8 +84,12 @@ protected virtual object CreateObjectForSerialization(IEnumerable<NodeSearchElem
         /// </summary>
         public static string GetFullyQualifiedName(NodeSearchElement element)
         {
+            if (element.ElementType.HasFlag(ElementTypes.Packaged) && element.ElementType.HasFlag(ElementTypes.BuiltIn))
```

Comment:
why not add this condition inside the if at line 88 ?
```
if (element.ElementType.HasFlag(ElementTypes.Packaged))
{
   if (element.ElementType.HasFlag(ElementTypes.BuiltIn) {
       return string.Format("{0}{1}.{2}", "bltinpkg://", element.FullCategoryName, element.Name);
    }
    //Use FullCategory and name as read from _customization.xml file
                return string.Format("{0}{1}.{2}", "pkg://", element.FullCategoryName, element.Name);
```

Reply:
I don't like using nested if statements if avoidable, if you much prefer your sample code will make the switch.